lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlockBlobFileSystemE2E.cs

```diff
@@ -138,6 +139,54 @@ public void TestDeleteE2E()
             Assert.False(CheckBlobExists(blob));
         }
 
+        [Fact(Skip = SkipMessage)]
+        public void TestGetChildBlobsInContainerE2E()
+        {
+            // setup
+            string[] parentFileNames = new string[] { "sample1", "sample2", "sample3" };
+            string[] folderfileNames = new string[] { "folder1/sample4", "folder1/sample5" };
+            string[] parentListNames = new string[] { "sample1", "sample2", "sample3", "folder1/" };
```

Comment:
These three arrays and their naming conventions make if very difficult to follow your logic. Can this be simplified with one list and the use of select statements? Something like the following. At the very least change parentFileNames to a concrete list so the concat at 150 isn't necessary. 
```csharp
string[] fileNames = new string[] { "sample1", "sample2", "sample3", "folder1/sample4", "folder1/sample5" };

var rootFilenames = fileNames.GroupBy(f => f.Split('/')[0]).Select(g => g.Key);
ValidateChildenWithBlobs(_container.Uri, rootFilenames);

var folderBlobs = fileNames.Where(f => f.StartsWith("folder1");
ValidateChildenWithBlobs(folderUri, folderBlobs);
```
OR
```csharp
string[] fileNames = new string[] { "sample1", "sample2", "sample3","folder1/sample4", "folder1/sample5" };
string[] expectedFolderChildren = new string[] { "folder1/sample4", "folder1/sample5" };
string[] expectedRootChildren = new string[] { "sample1", "sample2", "sample3", "folder1/" };
```